lib/sharing/editor_ui.dart

```diff
@@ -111,22 +119,20 @@ abstract class EditorUi {
     ga.sendEvent('main', 'run');
     runButton.disabled = true;
 
-    var compilationTimer = Stopwatch()..start();
-
+    final compilationTimer = Stopwatch()..start();
     final compileRequest = CompileRequest()..source = fullDartSource;
+    // If the null safety toggle has changed from the last execution to this
+    // one, destroy the frame.
+    final shouldDestroyFrame =
+        nullSafetyWasPreviouslyEnabled == !nullSafetyEnabled;
```

Comment:
nit: `nullSafetyWasPreviouslyEnabled != nullSafetyEnabled`

Reply:
`nullSafetyWasPreviouslyEnabled` starts out as `null`, which is why I intentionally went with the "not" operation. It doesn't matter too much. It just means that we would always destroy the initial iframe on the first executio, as `null` will always be not equal to `true` or `false`.